.github/workflows/go.yml

```
@@ -94,7 +100,7 @@ jobs:
           context: .
           push: false
           tags: ${{ steps.export_build_tag.outputs.docker_build_number }}
-          file: Dockerfile.drone
+          file: ${{ steps.check_files.outputs.files_exists == 'true' && "Dockerfile.release" || "Dockerfile.drone }}
```

Is the diff a bit interesting, or is there a missing end quote on the drone dockerfile?
```suggestion
          file: ${{ steps.check_files.outputs.files_exists == 'true' && "Dockerfile.release" || "Dockerfile.drone" }}
```